### PR TITLE
Misc. test improvements and other cleanup

### PR DESCRIPTION
**Pull request by ddabble**
*Depends on #350, and should change base to `dev` after that PR has been merged.*

Highlights:
* Reorganized news tests (7722cfd59f8a6be64dd5e21769ddc978224c4d5a and 895643cb2c7fc0f805760eb87f7f05f6a72cdb44)
* Added some `related_name`s (06679ebdbf78596a31a5f8e83bfa38d804eaff19)
* Change system accesses directly from the member modal (40f2282f341c4d0077aa99a31b2c541f84deb956)
* Added tests for all GET endpoints (eab90f7d53af4a8c10452afd8c7c3621a8033e6a)
* Misc. code cleanup (d34abe2d3e77ad550353193c7f108ddb5d81c6ec)

### CODE REVIEW

**Review by Maisergodt — APPROVED** [review]
Looks good to me! I don't have any disagreement with this PR, although should we review and merge #328 before this one?

File: news/tests/forms/test_time_place_form.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+        self.assertFalse(form.is_valid())
+
+    def test_time_place_form_data_with_end_time_none_is_not_valid(self):
+        form_data = self.valid_time_place_form_data
+        form_data["end_time"] = None
+
+        form = TimePlaceForm(data=form_data)
+
+        self.assertFalse(form.is_valid())
```
[the remark this answers is not in the document]
Comment (Fredrikaaa) [reply news/tests/forms/test_time_place_form.py:50]: Looks good

**Review by Fredrikaaa — APPROVED** [review]
Looks good to me. Codecov wants a few more tests, but it should be fine. :)

File: news/tests/forms/test_time_place_form.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (ddabble) [reply news/tests/forms/test_time_place_form.py:50]: @Maisergodt I think you might have overlooked something, as the file definitely has a newline at EOF, as far as I can tell 😅🤔

File: news/tests/views/test_article_views.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+        self.article = Article.objects.create(
+            title='PUBLISHED',
+            image=MOCK_JPG_FILE,
+            publication_time=timezone.localtime() - timedelta(days=1),
+        )
+
+    def test_admin(self):
+        response = self.client.get(reverse('admin-articles'))
+        self.assertNotEqual(response.status_code, 200)
```
[the remark this answers is not in the document]
Comment (ddabble) [reply news/tests/views/test_article_views.py:28]: Yes, nice suggestion! 😄👌

**Comment by ddabble** [comment]
@Maisergodt This PR and #328 don't seem to overlap too much, so I'd say it's perfectly fine to merge this PR first 🙂

**Review by Maisergodt — APPROVED** [review]
Sorry for having you wait this long. It is fine to merge this PR. I kind of rushed in the thought of merging this PR would also imply sequentially approving/merging the dependent PRs first. Which is absolutely not true, just some failed logic :)

**Comment by ddabble** [comment]
Hahah, understandable 😅 But yeah, no problem 😊

**Pull request merged**